books/hp-sorcerers-stone.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Harry Potter and The Sorcerers Stone",
+  "storyline": "Sixteen-year-old Clary Fray just wishes that her life would go back to normal. But what's normal when you're a demon-slaying Shadowhunter and you can suddenly see Downworlders like werewolves, vampires, and faeries? She continues trying to make sense of the swiftly changing events and relationships in her life as she becomes further involved with the Shadowhunters and their pursuit of demons and discovers some terrifying truths about her parents, her brother Jace, and her boyfriend Simon.",
+  "authors": ["J.K. Rowling"],
+  "genres": ["Fantasy Fiction, Young Adult Fiction"],
+  "characters": ["Harry Potter", "Albus Dumbledore", "Rubeus Hagrid", "Hermiona Granger", "Ron Weasley"],
+  "pages": 223,
+  "ISBN": "	0747532699",
+  "language": "english",
+  "publisher": "Bloomsbury",
+  "publication_date": "26-06-97"
```

Comment:
`publication_date` is incorrect. It should be in the following format:
https://github.com/404minds/books-db/blob/a49ed698b7a84e69f32d5e6a9c99c41b6f96cd25/books/1984.json#L20